src/commands/repos/UpdateRepos.ts

```diff
@@ -35,6 +38,62 @@ export class UpdateRepos extends AbstractReposCommand {
         return true;
     }
 
+    private moveNodeModules(repo: Repository): Promise<string> {
+        fs.move(path.join(repo.baseDir, 'node_modules'),
+                path.join(repo.baseDir, '__node_modules'), (err) => {
+                if (err) {
+                    console.error('Failed to move node_modules folder', err);
+                    return Promise.resolve('Failed to move node_modules folder', err);
+                }
+            });
+        return Promise.resolve('');
+    }
+
+    private restoreNodeModules(repo: Repository): Promise<void> {
+        fs.move(path.join(repo.baseDir, '__node_modules'),
+                path.join(repo.baseDir, 'node_modules'),
+                (err) => {
+                    if (err) {
+                        console.error('Failed to move node_modules folder', err);
+                        return Promise.reject('Failed to move node_modules folder', err);
+                    } else {
+                        return Promise.resolve();
+                    }
+                });
+    }
+
+    private isRestoreNodeModules(repo: Repository): boolean {
```

Comment:
rename to areNodeModulesCheckedIn -> if true remove __node_modules 
else restoreNodeModules